SchematicsAPI_Multi_NIC.md

```diff
@@ -0,0 +1,697 @@
+## Schematics API - Multi NIC F5 Provisioning
+
+This document provides the steps to provision multi NIC using F5  IBM Schematics API Curl Commands .
+
+### 1.	Create tokens:
+access_token and refresh_token need to be created first.
+
+Command to create them is 
+
+<pre><code>export IBMCLOUD_API_KEY=&lt;ibmcloud_api_key>
+curl -X POST "https://iam.cloud.ibm.com/identity/token" -H "Content-Type: application/x-www-form-urlencoded" -d "grant_type=urn:ibm:params:oauth:grant-type:apikey&apikey=$IBMCLOUD_API_KEY" -u bx:bx
+</code></pre>
+
+The "access_token" is considered as iam_token or bearer token. It can be used by appending "Bearer" keyword before it.
+
+<pre><code>Example: 
+
+curl -X POST "https://iam.cloud.ibm.com/identity/token" -H "Content-Type: application/x-www-form-urlencoded" -d "grant_type=urn:ibm:params:oauth:grant-type:apikey&apikey=$IBMCLOUD_API_KEY" -u bx:bx | jq
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100  2469  100  2368  100   101    380     16  0:00:06  0:00:06 --:--:--   607
+{
+  "access_token": "eyJraxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+  "refresh_token": "OKDSlxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+  "ims_user_id": 8009626,
+  "token_type": "Bearer",
+  "expires_in": 3600,
+  "expiration": 1593170371,
+  "scope": "ibm openid"
+} 
+</code></pre>
+
+### 2.  Export the access_token and refresh_token obtained in step 1 as environment variables as ACCESS_TOKEN and REFRESH_TOKEN respectively.
+
+<pre><code>export ACCESS_TOKEN=&lt;access_token>
+export REFRESH_TOKEN=&lt;refresh_token>
+</code></pre>
+
+
+### 3.  Create workspace.
+
+ Create a schematic workspace in Frankfurt as show below,
+ <pre><code>curl --request POST --url https://eu-de.schematics.cloud.ibm.com/v1/workspaces -H "Authorization: Bearer &lt;access_token>" -d '{"name":"<workspace_name>","type": ["terraform_v0.12"],"description": "<workspace_description>","resource_group": "<resource_group_ID>","tags": [],"template_repo": {"url": "<source_repo_url>"},"template_data": [{"folder": ".","type": "terraform_v0.12","variablestore": [{"name": "variable_name1","value": "variable_value1"},{"name": "variable_name2","value": "variable_value2"}]}]}'
+</code></pre>
+
+
```

Comment:
Need to add the text "Example" here. Also please mention the use of request.json file to save the data parameters for creating the workspace.